cosmic-update.jenkinsfile

```diff
@@ -0,0 +1,147 @@
+import groovy.json.JsonSlurper
+def currentRelease
+def previousRelease
```

Comment:
Actually, we've switched the Release pipeline JFs to use the shared groovy library. This allows the current/previous release versions to be retrieved within a stage so 'executing from stage' functionality actually works.

[Update-stable-ids Jenkinsfile](https://github.com/reactome/release-update-stable-ids/blob/develop/Jenkinsfile)

Reply:
This file predates the shared library. I checked it in yesterday, but I think I authored it... a while ago! I'll make some changes to be more consistent with the other Jenkinsfiles.